test/csuite/timestamp_abort/smoke.sh

```diff
@@ -33,6 +33,9 @@ $TEST_WRAPPER $test_bin -m $default_test_args -c
 #$TEST_WRAPPER $test_bin -m $default_test_args -L
 $TEST_WRAPPER $test_bin -C $default_test_args
 $TEST_WRAPPER $test_bin -C $default_test_args -c
-$TEST_WRAPPER $test_bin -C $default_test_args -B -I 3
 $TEST_WRAPPER $test_bin -C -m $default_test_args
 $TEST_WRAPPER $test_bin -C -m $default_test_args -c
+
+# Don't test backups in the compatibility mode. MongoDB no longer uses it, and the compatibility
+# mode is inherently incompatible with backup-related fixes that add new log records.
+#$TEST_WRAPPER $test_bin -C $default_test_args -B -I 3
```

Comment:
I agree this is the first item to change and it solves the testing problem immediately.